### ISSUE TITLE
What about ff99-disp

### ISSUE THREAD

**Issue by bcossins** [OpenBioSim/biosimspace#346]
Only me.

**Is your feature request related to a problem? Please describe.**
Saw Julien publishing on simulations with a99sb-disp - https://github.com/paulrobustelli/Force-Fields 
This ff is not currently in BioSimSpace is it?
I would like to use this.

**Describe the solution you'd like**
Can we add this ff to the others that are in there?
Is there an easy way for users to add new protein ffs?

**Describe alternatives you've considered**

**Additional context**

**Comment by lohedges** [comment]
I think we'd just need a way to make it easier to parameterise with custom force fields. We couldn't support this easily as is since it's just a GitHub repo with the code, not a package that we could add as a requirement. Assuming that the force field doesn't contain any unsupported terms, e.g. CMAP, then it should be easily enough to add something that can find the force field files by name given some path (or if you've added them to AMBER HOME).

**Comment by bcossins** [comment]
Its not in the std amber ffs but is freely available from the linked repo
and widely used.
It requires a modified tip4p too I think.

**Comment by lohedges** [comment]
Yeah, it seems self contained. My issue is that it's not versioned so I'm not going to write an API on the basis of something that we can't constrain and might change in future without us realising. I'd rather have a general way for a user to use fully custom AMBER parameters, e.g. by sticking them in a directory then searching for them by the specified forcefield name. It might be tricky if it requires custom waters too, since we don't solvate with AMBER. (They do provide GROMACS files, though.)

**Comment by bcossins** [comment]
I dont think this one is going to change cos its not in the core of amber.
There are lots of DEShaw and other varients around though all useful for
slightly different things.

The water model is just adjusted tip4p so can be added with any tool.
Amber/gromacs.

**Comment by lohedges** [comment]
Just to say that the [code]](https://github.com/paulrobustelli/Force-Fields) has no license, so there's no way that we could vendor it, or easily include as a dependency. I'm sure this isn't an issue in practice, so might be worth asking them to add one. I'll think about a more general interface, though, since I'd rather support _any_ custom force field parameters rather than these specific ones.

**Comment by bcossins** [comment]
Yes, there are quite a lot of modified amber ffs that would be useful to
use. They all exist as datasets being handed between academics tbh

A way to add them to bss/sire would be handy.
